src/raftstore/store/snap.rs

```diff
@@ -31,17 +31,13 @@ use kvproto::raft_serverpb::RaftSnapshotData;
 
 use raftstore::Result as RaftStoreResult;
 use raftstore::store::Msg;
-use storage::{CfName, CF_DEFAULT, CF_LOCK, CF_WRITE};
 use util::transport::SendCh;
 use util::HandyRwLock;
 use util::collections::{HashMap, HashMapEntry as Entry};
 
 use super::engine::Snapshot as DbSnapshot;
 use super::peer_storage::JOB_STATUS_CANCELLING;
 
-// Data in CF_RAFT should be excluded for a snapshot.
-pub const SNAPSHOT_CFS: &'static [CfName] = &[CF_DEFAULT, CF_LOCK, CF_WRITE];
```

Comment:
I still prefer using snapshot CFS to make the code more clear.

If we add another CF later, we don't need change here.